MPF.Modules/BaseParameters.cs

```diff
@@ -1849,9 +1849,8 @@ protected static string GetPlayStation5Version(char? driveLetter)
                     case 'A': return Region.Asia;
                     case 'C': return Region.China;
                     case 'E': return Region.Europe;
-                    case 'J': return Region.JapanKorea;
                     case 'K': return Region.SouthKorea;
-                    case 'P': return Region.Japan;
+                    case 'P': return null; // P-region discs may be Japan, Asia or Korea
```

Comment:
It sounds like a more accurate thing would be to return `Region.Asia` instead, then.

Reply:
Thank you. Solving issues exactly as requested is not always the right course of action :)

Reply:
Even if it's not always correct, I'd rather SLPM return Japan, then Asia.  It is much more common to be Japan.

Now, if you want to get a bit more specific, I could list out the number ranges that are likely to be certain regions.  But if you don't want to deal with that, that's ok.